packages/client/lib/service/txpool.ts

```diff
@@ -75,46 +75,51 @@ export class TxPool {
    * @param  peer peer
    */
   async announced(txHashes: Buffer[], peer: Peer) {
-    if (txHashes.length) {
-      //this.config.logger.info(`Tx pool: received new pooled tx hashes number=${txHashes.length}`)
-
-      const reqHashes = []
-      for (const txHash of txHashes) {
-        const txHashStr = txHash.toString('hex')
-        if (txHashStr in this.pending || txHashStr in this.handled) {
-          continue
-        }
-        reqHashes.push(txHash)
+    if (txHashes.length === 0) {
+      return
+    }
+    this.config.logger.info(`TxPool: received new pooled hashes number=${txHashes.length}`)
```

Comment:
Ok, after finishing this is just nothing we want on the `info` level I guess since it is too verbose. We might just move to `debug`.